### PR TITLE
Do not warn when http source uses `file:///` URI scheme

### PR DESCRIPTION
**Pull request by dnkoutso**
closes https://github.com/CocoaPods/CocoaPods/issues/7460

### CODE REVIEW

File: lib/cocoapods/installer/pod_source_installer.rb [hunk 1]
```diff
@@ -133,8 +133,7 @@ def download_source
       def verify_source_is_secure(root_spec)
         return if root_spec.source.nil? || root_spec.source[:http].nil?
         http_source = root_spec.source[:http]
-        return if http_source.downcase.start_with?('https://')
-
+        return if http_source.downcase.start_with?('https://', 'file:///')
```
Comment (segiddins) [inline lib/cocoapods/installer/pod_source_installer.rb]: We should parse the URI and check the .scheme property, instead of just doing a string comparison

Comment (dnkoutso) [reply lib/cocoapods/installer/pod_source_installer.rb]: will update

Comment (dnkoutso) [reply lib/cocoapods/installer/pod_source_installer.rb]: updated to use `URI` and check the `scheme`

**Comment by dnkoutso** [comment]
le green

**Pull request merged**